test/framework/k2s/addons.go

```diff
@@ -157,3 +158,28 @@ func GetImagesForAddon(addon Addon) ([]string, error) {
 
 	return lo.Union(images), nil
 }
+
+type AddonsInfo struct {
```

Comment:
I would vote for putting the types all at the top of the file. At least put them together :-).